docs/terms/execution_engine.md

```diff
@@ -70,13 +70,13 @@ If you are interested in using and accessing data with an Execution Engine that
 
 - `name`
 - `caching`
-- `batch_spec_defaults` (is this needed?)
+- `batch_spec_defaults`
```

Comment:
Just checking - should these changes be in this PR?

Reply:
The original Jira ticket included a couple more pages that contained the reference we were removing.  Those pages were out dated and could be removed entirely (This was obviated by another PR).  Along with the instruction to remove those pages, there was a note that `execution_engine.md` contained content derived from those pages, and that while it did not contain the reference we were removing there were some notes (these ones, specifically) which should be removed.  I went ahead and did that as part of "cleanup" for the removing of those other pages, and felt it fell under the mandate of this PR -- it just isn't as apparent now since those removed files ended up being removed by another PR before this one was approved.

Reply:
tl;dr yes, these changes were intended for this PR.